CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog for the Mapbox Search SDK for Android
 
+## 1+
```

Comment:
Is it going to be GA version (not rc) of the search SDK?

Reply:
I don't have a definitive answer. The plan is to maintain two release-like branches concurrently: 1.x and 2.x

Reply:
Regardless of whether it's version 1.x, 2.x, or even 10.x, it's important to consider whether the SDK is published as a General Availability (GA) release, without 'alpha', 'beta', or 'rc' in the version name. If you're confident that the API is completely stable and won't undergo any breaking changes during the 1.x cycle, feel free to publish it as a GA release. Otherwise, it makes sense to remain in the 'rc' phase for a while.

cc @mapbox-danny